lib/ember-app.js

```diff
@@ -153,7 +153,13 @@ function serializeHTML(doc, rootElement) {
 function buildWhitelistedRequire(whitelist, distPath) {
   return function(moduleName) {
     if (whitelist.indexOf(moduleName) > -1) {
-      return require(path.join(distPath, 'node_modules', moduleName));
+      try {
+        // Attempt to see if it's a built-in module
+        return require(moduleName);
+      } catch (e) {
+        // See if it's an npm module, since it wasn't built in
+        return require(path.join(distPath, 'node_modules', moduleName));
```

Comment:
`path.resolve` is called here: https://github.com/ember-fastboot/ember-fastboot-server/blob/master/lib/server.js#L53